### ISSUE TITLE
Attempt to invoke virtual method 'void androidx.swiperefreshlayout.widget.SwipeRefreshLayout.setVisibility(int)' on a null object reference

### ISSUE THREAD

**Issue by skjnldsv** [nextcloud/android#7228]
************ CAUSE OF ERROR ************

```java
java.lang.NullPointerException: Attempt to invoke virtual method 'void androidx.swiperefreshlayout.widget.SwipeRefreshLayout.setVisibility(int)' on a null object reference
	at com.owncloud.android.ui.fragment.FileDetailActivitiesFragment.populateList(FileDetailActivitiesFragment.java:415)
	at com.owncloud.android.ui.fragment.FileDetailActivitiesFragment.lambda$null$3$FileDetailActivitiesFragment(FileDetailActivitiesFragment.java:370)
	at com.owncloud.android.ui.fragment.-$$Lambda$FileDetailActivitiesFragment$ZzLUj33qKg0e1Ki8Vd5bki8I6B4.run(Unknown Source:6)
	at android.os.Handler.handleCallback(Handler.java:883)
	at android.os.Handler.dispatchMessage(Handler.java:100)
	at android.os.Looper.loop(Looper.java:214)
	at android.app.ActivityThread.main(ActivityThread.java:7403)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:935)
```

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30140051
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: xiaomi
Device: jasmine_sprout
Model: Mi A2
Id: QKQ1.190910.002
Product: jasmine

************ FIRMWARE ************
SDK: 29
Release: 10
Incremental: V11.0.12.0.QDIMIXM

**Comment by tobiasKaminsky** [comment]
fixed via #7105 or #7212

**Comment by AndyScherzinger** [comment]
Duplciate of #7188
